backend/alembic/versions/20260425_0016_add_market_to_market_breadth.py

```
@@ -0,0 +1,94 @@
+    inspector = sa.inspect(bind)
+    existing_uniques = {uc["name"] for uc in inspector.get_unique_constraints("market_breadth")}
+
+    if dialect == "sqlite":
+        with op.batch_alter_table("market_breadth") as batch_op:
+            # Try all plausible names for the date-only unique; ignore if none exist.
+            for name in ("market_breadth_date_key", "uq_market_breadth_date"):
+                try:
+                    batch_op.drop_constraint(name, type_="unique")
+                    break
+                except Exception:
+                    continue
```

Ruff (0.15.11)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)